app/containers/HomePage/index.js

```diff
@@ -77,21 +59,29 @@ export class HomePage extends React.PureComponent { // eslint-disable-line react
               <FormattedMessage {...messages.trymeHeader} />
             </H2>
             <Form onSubmit={this.props.onSubmitForm}>
-              <label htmlFor="username">
-                <FormattedMessage {...messages.trymeMessage} />
-                <AtPrefix>
-                  <FormattedMessage {...messages.trymeAtPrefix} />
-                </AtPrefix>
-                <Input
-                  id="username"
-                  type="text"
-                  placeholder="mxstbr"
-                  value={this.props.username}
-                  onChange={this.props.onChangeUsername}
+              <div>
+                <label htmlFor="username">
+                  <FormattedMessage {...messages.trymeMessage} />
+                  <AtPrefix>
+                    <FormattedMessage {...messages.trymeAtPrefix} />
+                  </AtPrefix>
+                  <Input
+                    id="username"
+                    type="text"
+                    placeholder="mxstbr"
+                    value={this.props.username}
+                    onChange={this.props.onChangeUsername}
+                  />
+                </label>
+                <FormattedMessage
+                  {...messages.userLink}
+                  values={{
+                    link: (<Link to={`/users/${this.props.username}`}>{this.props.username}</Link>),
+                  }}
```

Comment:
This user link needs to have some sort of design, right now it's a bit ugly in the middle of the page.